bundles/mapping/mapmodule_ol3/plugin/controls/ControlsPlugin.js

```diff
@@ -108,7 +112,36 @@ Oskari.clazz.define(
                 'DisableMapMouseMovementRequest' : mapMovementHandler
             };
         },
+        disableMouseDragZoom: function () {
+            var me = this;
+            if ( this.boxZoom ) {
+                this.getMap().removeInteraction( this.boxZoom );
+            }
+            if ( this.removedInteractions ) {
+                this.removedInteractions.forEach( function ( interaction ) {
+                    me.getMap().addInteraction( interaction );
+                });
```

Comment:
You should reset the removedInteractions list here. Otherwise it will just keep getting more and more interaction references.